docs-web/src/main/webapp/src/partial/docs/document.view.html

```diff
@@ -100,16 +100,65 @@ <h1>
           <span class="fas fa-random"></span>
           {{ 'document.view.workflow_current' | translate }}
         </p>
+        <style>
+          /* Rating Code adapted and sourced from https://codepen.io/hesguru/pen/BaybqXv */
 
+          .ratings-stars {
+            height: 46px;
+            padding: 0 10px;
+            text-align: center;
+          }
+          .ratings-stars:not(:checked) > input {
+            position: absolute;
+            top: -9999px;
+          }
+          .ratings-stars:not(:checked) > label {
+            float: right;
+            width: 1em;
+            overflow: hidden;
+            white-space: nowrap;
+            cursor: pointer;
+            font-size: 20px;
+            color: #ddd;
+          }
+          .ratings-stars:not(:checked) > label:before {
+            content: "★ ";
+          }
+          .ratings-stars > input:checked ~ label {
+            color: #5bc0de;
+          }
+          .ratings-stars:not(:checked) > label:hover,
+          .ratings-stars:not(:checked) > label:hover ~ label {
+            color: #46b8da;
+          }
+        </style>
         <div class="text-center card">
           <p>{{ document.route_step.name }}</p>
           <p>
             <span class="fas fa-exchange-alt" ng-if="document.route_step.type == 'APPROVE'"></span>
             <span class="fas fa-check-circle" ng-if="document.route_step.type == 'VALIDATE'"></span>
+            <span class="far fa-file" ng-if="document.route_step.type == 'RESUME_REVIEW'"></span>
             {{ 'workflow_type.' + document.route_step.type | translate }}
             <span class="label label-default"><acl data="document.route_step.target"></acl></span>
           </p>
+
           <p ng-show="document.route_step.transitionable">
+            <div ng-show="document.route_step.type == 'RESUME_REVIEW'">
+              <div class="ratings-stars" ng-repeat="category in ratingCategories">
+                <strong class="rating">{{ category.name }}</strong><br>
+                <input type="radio" id="-{{ category.id }}star5" name="{{ category.id }}" value="5" ng-model="category.value">
```

Comment:
The “5 stars” button doesn’t work because of this typo:


```suggestion
                <input type="radio" id="star5-{{ category.id }}" name="{{ category.id }}" value="5" ng-model="category.value">
```